mesh-worker-service/src/main/java/io/functionmesh/compute/util/FunctionsUtil.java

```diff
@@ -185,6 +185,9 @@ public static V1alpha1Function createV1alpha1FunctionFromFunctionConfig(String k
         v1alpha1FunctionSpec.setMaxPendingAsyncRequests(functionConfig.getMaxPendingAsyncRequests());
 
         v1alpha1FunctionSpec.setReplicas(functionDetails.getParallelism());
+        if (customRuntimeOptions.getMaxReplicas() > functionDetails.getParallelism()) {
```

Comment:
there should be a check(maybe in the UI ) to make sure the maxReplicas is greater than the replicas.